lib/tests/streamlit/type_util_test.py

```diff
@@ -91,9 +96,66 @@ def test_to_bytes(self):
         with self.assertRaises(RuntimeError):
             to_bytes(string_obj)
 
-    def test_data_frame_to_bytes_numpy_dtype_exception(self):
+    def test_data_frame_with_dtype_values_to_bytes(self):
         df1 = pd.DataFrame(["foo", "bar"])
         df2 = pd.DataFrame(df1.dtypes)
 
-        with self.assertRaises(NumpyDtypeException):
+        try:
             data_frame_to_bytes(df2)
+        except Exception as ex:
+            self.fail(f"Converting dtype dataframes to Arrow should not fail: {ex}")
+
+    def test_fix_unsupported_column_types(self):
```

Comment:
Can you add a short docstring to this and the next test explaining what they test for?

Reply:
Added 👍